common/calibration-model.cpp

```diff
@@ -146,8 +146,8 @@ void calibration_model::update(ux_window& window, std::string& error_message)
         to_open = false;
     }
 
-    auto table = (librealsense::ds::coefficients_table*)_calibration.data();
-    auto orig_table = (librealsense::ds::coefficients_table*)_original.data();
+    auto table = (librealsense::ds::ds5_coefficients_table*)_calibration.data();
+    auto orig_table = (librealsense::ds::ds5_coefficients_table*)_original.data();
```

Comment:
Can you explain?
A common methid use `ds5_` structure while ds6 use `ds6_` structure?

Reply:
This calibration model is not enabled yet - see method:
bool calibration_model::supports()
{
    bool is_d400 = dev.supports(RS2_CAMERA_INFO_PRODUCT_LINE) ?
        std::string(dev.get_info(RS2_CAMERA_INFO_PRODUCT_LINE)) == "D400" : false;

    return dev.is<rs2::auto_calibrated_device>() && is_d400;
}

If we want it to be enabled, we have still work to do

Reply:
We can handle it later on..